ui/coordinateTrainer/src/ctrl.ts

```diff
@@ -179,7 +224,7 @@ export default class CoordinateTrainerCtrl {
       this.keyboardInput.value = '';
     }
 
-    if (this.isAuth) {
+    if (this.isAuth && this.timeControlChoice == 'thirtySeconds') {
```

Comment:
And do not push on server.

Reply:
Ah, I actually meant only provide two time controls, 30s and infinite. This is an interesting solution though. I'm just thinking that the user will be confused when their score is recorded for 30s, they switch to 1min, play another round, and then notice their scores stop being saved.

Reply:
That's definitely confusing. One timed option seems good enough for a start. If you want to add more, it definitely should store the results as well and separately.